test/C/DistributedCountDecentralizedLate.lf

```diff
@@ -1,38 +1,43 @@
-/** 
+ /** 
  * Test a form of a distributed deterministic system
  * where a federate that receives timestamped messages has a timer in addition to the messages
  * as triggers. Therefore, careful coordination of the advancement of time using Ptides is needed.
  * @author Edward A. Lee
  * @author Soroush Bateni
  */
 target C {
-    timeout: 5 sec,
+    timeout: 4900 msec,
```

Comment:
By switching back to `5 sec`, an extra message will be sent by `Count` at `(5 sec, 0)`. If it is on-time, it is going to be seen because shutdown events happen at `(5 sec, 0)`. If the message is tardy, it is not going to be seen. My fear is that this could lead to the test failing non-deterministically.